src/Symfony/Component/HttpFoundation/Tests/ResponseTest.php

```diff
@@ -369,6 +369,12 @@ public function testExpire()
         $this->assertNull($response->headers->get('Expires'), '->expire() removes the Expires header when the response is fresh');
     }
 
+    public function testNullExpireHeader()
```

Comment:
Think, you'd better place this test into testExpire() method (see line 339 in this file) and use more consistent format, like this:
```php
$response = new Response();
$response->headers->set('Expires', null);
$response->expire();
$this->assertNull($response->headers->get('Expires'), 'Place description of expected behavior here')
```

Reply:
I thought about it, but this bug is not about calling `expire` on the response, actually this call is not necessary at all. That's why I've decided against that, in order to keep the test minimal.